docs/develop/convert-javascript-to-typescript.md

```diff
@@ -48,25 +48,12 @@ You can use the Office Add-in template in Visual Studio to create an add-in that
 
 4. From the **Tools** tab, choose **NuGet Package Manager** and then select **Manage NuGet Packages for Solution...**.
 
-5. With the **Browse** tab selected, enter **office-js.TypeScript.DefinitelyTyped** into the search box. Install or update this package if it is already installed. This will add the TypeScript type definitions for the Office.js library to your project.
-
-6. In the same search box, enter **jquery.TypeScript.DefinitelyTyped**. Install or update this package if it is already installed. This will add the jQuery TypeScript definitions into your project. The packages for both jQuery and Office.js will now appear in a new file generated by Visual Studio, called **packages.config**.
+5. In the same search box, enter **jquery.TypeScript.DefinitelyTyped**. Install or update this package if it is already installed. This will add the jQuery TypeScript definitions into your project. The packages for both jQuery and Office.js will now appear in a new file generated by Visual Studio, called **packages.config**.
```

Comment:
```suggestion
5. With the **Browse** tab selected, enter **jquery.TypeScript.DefinitelyTyped**. Install this package, or update it if it's already installed. This will ensure the jQuery TypeScript definitions are included in your project. The packages for jQuery appear in a file generated by Visual Studio, called **packages.config**.
```